src/orca-log-catcher/OrcaLogger.res

```diff
@@ -421,8 +424,28 @@ let make = (
     metadata := value
   }
 
+  let calculateAndUpdateCounterHook = eventName => {
+    let updatedCounter =
+      1 +
+      switch eventsCounter.contents->Js.Dict.get(eventName) {
```

Comment:
Could you please move this switch in a separate variable to improve code readability